src/components/App/App.css

```diff
@@ -3,6 +3,10 @@ main {
   width: 100vw;
 }
 
+h1 {
+  color: #ad9976;
```

Comment:
I'm anticipating a merge conflict here as I also worked on the title. I'll be adding just some positioning to the h1 fyi